### DISCUSSION TITLE
메인화면에서 이미지가 크게 보이는 현상

### DISCUSSION THREAD

**Discussion by jiwon1027** [pronist/hELLO#126] [Q&A] [answered]
안녕하세요! 이번에 hello 스킨을 알게되어서 적용해봤는데 몇가지 오류가 있어서 discussion 남깁니다!
![image](https://user-images.githubusercontent.com/68945145/185739038-1b6c2a30-7813-43eb-a92f-ba10dcb4ed4a.png)

사진과 같이 게시물안에 올린 사진이 게시물보다 위에 뜨게 되고 사진이 매우 크게나온 현상입니다.

어떻게 하면 좋을지 도움을 얻고싶습니다. 감사합니다!

version : 3.6.2, 3.6.1 모두 사용해봤습니다
os : Windows10

https://ddangjiwon.tistory.com/

**Comment by pronist** [comment, marked as answer]
@jiwon1027 안녕하세요!

**스킨 편집**에서 커버를 설정하시거나 **글 목록 스타일**을 변경하시면 되겠습니다. :)

**Reply by jiwon1027** [reply]
정말 감사합니다! 스킨 이쁘게 잘 쓰겠습니다!!
